src/main/scala/ScalaLirc.scala

```diff
@@ -65,12 +29,12 @@ class ScalaLirc(sourceFile: String = "/etc/lirc/lircd.conf") {
       try {
         command ! match {
           case 0 => true
-          case error if error > 0 => 
+          case error if error > 0 =>
             System.err.println(s"Could not run `$command`")
             false
         }
       } catch {
-        case e: IOException => 
+        case e: IOException =>
```

Comment:
Hah I guess stripping the whitespace at the end of the lines that I added to vim actually works. PR looks ugly but it's for the better